packages/lib-classifier/src/plugins/tasks/DrawingTask/components/tools/DrawingToolRoot.js

```diff
@@ -1,56 +1,100 @@
 import PropTypes from 'prop-types'
-import React, { useState } from 'react'
+import React, { forwardRef, useState } from 'react'
+import styled from 'styled-components'
+import draggable from '../components/draggable'
 
-const STROKE_WIDTH = 1.5
-const SELECTED_STROKE_WIDTH = 2.5
+const STROKE_WIDTH = 2
+const SELECTED_STROKE_WIDTH = 3
 
-function DrawingToolRoot ({ children, tool }) {
-  const [ active, setActive ] = useState(true)
+const StyledGroup = styled('g')`
+  :focus {
+    outline: 1px solid black;
+  }
+`
+
+const DrawingToolRoot = forwardRef(({
+    children,
+    isActive,
+    label,
+    mark,
+    onDelete,
+    onDeselect,
+    onSelect,
+    svg,
+    tool
+  }, ref) => {
+  console.log('rendering', label)
+  const [ active, setActive ] = useState(isActive)
   const mainStyle = {
     color: tool && tool.color ? tool.color : 'green',
     fill: 'transparent',
     stroke: tool && tool.color ? tool.color : 'green',
-    strokeWidth: active ? SELECTED_STROKE_WIDTH : STROKE_WIDTH
   }
 
-  function onPointerDown (event) {
-    console.log(`${tool.type} clicked`)
-    event.stopPropagation()
-    event.preventDefault()
-    setActive(true)
+  function onKeyDown (event) {
+    switch (event.key) {
+      case 'Backspace': {
+        event.preventDefault()
+        event.stopPropagation()
+        onDelete(mark)
+        return false
+      }
+      default: {
+        return true
+      }
+    }
   }
 
-  function onPointerMove (event) {
-    console.log(`${tool.type} moved`)
-    event.stopPropagation()
-    event.preventDefault()
+  function select (event) {
+    if (!active) {
+      setActive(true)
+      onSelect(mark)
+      console.log('selecting', label)
+      console.log(document.activeElement)
```

Comment:
Leftover console logs from testing?

Reply:
Debugging why dragging doesn't start when you select a mark. The commit is marked WIP.